### DISCUSSION TITLE
@galacean/effects 与 galacean engine 这两是两个独立的东西吗？

### DISCUSSION THREAD

**Discussion by BUPTlhuanyu** [galacean/effects-runtime#776] [General]
从代码看 @galacean/effects 与 galacean engine 是两个没有依赖的代码库，为啥会是两个独立的东西？我们项目中的 threejs 的游戏后面迁移用 effects 还是 engine？

**Comment by yiiqii** [comment]
两个是独立的，因为面向的场景不同，不过可以在 engine 中使用 effects，使用 https://github.com/galacean/engine-effects

**Reply by BUPTlhuanyu** [reply]
effects 加上 spine 插件加一起的包体积有1.78M，gzip之后176kb，这个功能模块能再拆细一点独立成包吗？

**Reply by yiiqii** [reply]
effects 已经把非核心的功能都抽成独立插件了 https://github.com/galacean/effects-runtime/tree/main/plugin-packages，目前主包相对确实较大，这块仔细梳理过，优化空间也有限，后面也关注看看如何再优化。
另外，spine-core 本身也比较大
